pkg/core/runtime/component/resilient.go

```diff
@@ -0,0 +1,65 @@
+package component
+
+import (
+	"time"
+
+	k8s_manager "sigs.k8s.io/controller-runtime/pkg/manager"
+
+	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
+)
+
+const (
+	backoffTime = 5 * time.Second
+)
+
+type ComponentFactory func(log logr.Logger) k8s_manager.Runnable
```

Comment:
I use `component.Manager` to operate with multiple `Component` as with a single one. After your LB changes Manager doesn't implement `Component` interface anymore. That's why I changed the interface to `Runnable` just to make it work.

Probably I have to create a new entity `CompositeComponent` and pass it to ResilientComponent, instead of passing Manager.